utils/run_tests.sh

```diff
@@ -3,29 +3,37 @@
 function wait_for_start {
     (
     HOST=${1}
-    PORT=${2}
+    STANDBY=${2}
+    PORT=${3}
     # Wait for stardog to be running
-    RC=1
     COUNT=0
     set +e
-    while [[ ${RC} -ne 0 ]];
+    while :
```

Comment:
this is strange syntax, I guess it is legal, but I hate it anyway cause it makes me google things. What if instead you change to 
```
not_all_nodes_found=true
while $not_all_nodes_found
do 
         # all nodes found yay!
         not_all_nodes_found=false
```

Reply:
Although I don't consider `while :` a strange syntax, I did address this somehow, but note that changing the expression for `$not_all_nodes_found` (or similar) might not be correct. in this case we are waiting for 2 things:
main cluster to be up and standby node to be up. I modified it a bit so it's more clear.
I am also breaking up the loop when everything is ready:
` if [[ $number_of_nodes -eq 2 && $RC -eq 0 ]]; then break; fi`
so I don't think it's necessary to change the value for the condition.